src/mds/CInode.cc

```diff
@@ -3765,14 +3757,23 @@ void CInode::validate_disk_state(CInode::validated_data *results,
       // set up basic result reporting and make sure we got the data
       results->performed_validation = true; // at least, some of it!
       results->backtrace.checked = true;
-      results->backtrace.passed = false; // we'll set it true if we make it
```

Comment:
We still need to start this at false or we'll have an undetermined result if we fail the backtrace check.